tests/reading-tests.md

```diff
@@ -0,0 +1,397 @@
+1. The line continues, chaining on `.toEqual(...)`
+   - In the first test, the argument was `'Fizz'`
+   - In this second test, the argument is `'Buzz'`
+
+To summarize, this second test expects the value of `fizzBuzz(25)` to equal `'Buzz'`.
+
+## Example on Replit.com
+
+We can run a version of this test suite on [this Replit code snippet](https://replit.com/@adacore/FizzBuzz-With-Tests#index.js).
+
+Replit.com supports Jest unit tests, although it has its own configurations to deal with. Let's purposely skip over those details in order to focus on the tests.
+
+The tests in [this REPL](https://replit.com/@adacore/FizzBuzz-With-Tests#index.js) are already set up.
+
+Navigate to the "Unit Tests" section, listed on the left-hand navigation menu.
+
+![Replit UI that highlights the "Unit Tests" tab on the left-hand navigation bar](../assets/tests_tests_replit-highlight-test-nav.png)
+
+The left pane now shows details about the existing test suite, with a list of existing tests.
+
+![Replit UI that highlights a list of four unit tests relating to FizzBuzz](../assets/tests_tests_replit-test-list.png)
+
+We can click into one of these tests and observe the contents of the test.
+
+![Modal that shows the details of the unit test titled `multiples-of-three`, including its test contents](../assets/tests_tests_replit-test-detail.png)
+
+We can run the tests using the "Run tests" button.
+
+Hopefully we see one failing test.
+
+![Replit UI featuring test failure messages in the right-hand console](../assets/tests_tests_replit-test-failure.png)
+
+We can use the error messages to change our code and get the test to pass!
+
+## Reading Jest Error Messages
+
+Leila and Nadia need to read through the test failure and both get aligned with which test is failing, and how it's failing.
+
+Consider this test failure message:
+
+```
+❌   non-multiples
+Message: The function did not return the number itself on numbers that are not multiples of three or five
+Stack Trace:
+Error: expect(received).toEqual(expected)
+
+Expected value to equal:
+  4
+Received:
+  undefined
+
+Difference:
+
+  Comparing two different types of values. Expected number but received undefined.
+```
+
+| <div style="min-width:200px;"> Piece of Output </div> | Notes                                                                                                                                                                                                   |
+| ----------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `❌ non-multiples`                                    | This indicates that a test by the name or description of `non-multiples` is failing                                                                                                                     |
```

Comment:
```suggestion
| `❌ not-multiple-of-three-or-five`                                    | This indicates that a test by the name or description of `not-multiple-of-three-or-five` is failing                                                                                                                     |
```
The name of this test being so short is an anti-pattern, in my mind. I offer this change because I know you feel strongly about not presenting anti-patterns to students in the curriculum.

Reply:
This is one of several changes that'd need to be made to fully carryout this change (including screenshots and changes to the repl itself)

Reply:
I'm gonna have to rework the whole replit situation-- during that time, I'll do a pass-through and update this test to this suggestion!